src/_alfasim_sdk/alfacase/generate_case_description_docstring.py

```diff
@@ -83,7 +111,21 @@ def _get_declaration(
             for predicate_function, handle_function in list_of_predicate_and_handles:
                 if predicate_function(value.type):
                     attribute_value = handle_function(value.type)
-                    lines.append(f"{BASE_INDENT}{attribute_name}: {attribute_value}")
+                    default_value = _get_default_value_of_case(value)
+
+                    # For CaseDescription, if the attributes has a default value, the empty space is not necessary
+                    if default_value:
+                        attribute_value = attribute_value.replace("|space|", "")
```

Comment:
What is the |space| for?

Reply:
code-block doesn't accept links or references, so I had to uses `literal-block` with " ` " characters
This brings some problems when the line ends with these special characters because sphinx doesn't insert a line break (I don't know the reason 🤷 )
My workaround solution was to add a space, but a normal space didn't work because pycharm was removing it 🤦 
So the solution was to uses a Unicode character that represents the space character.
The `| |` is just a way in sphinx re-use a text

Below is the docstring of `_get_class_with_reference` were wrote a docstring about this.
What your thoughts on this, this docstring is good enough?

```python
    :param bool add_space:
        inform if the name generated will need to have an unicode space at the end (generate by |space|),
        this is necessary when the class is the only reference of an attribute causing this line
        to be mixed with the next line over the documentation.

Reply:
Thanks for the explanation, but where do you get `'` from, from the default values? Can't we replace them by `"` instead during the default values generation? If that works, seems like a better solution to me as that would be a localized "hack" rather than something that spills through many parts of the code.

Reply:
`_get_class_with_reference` is not even called directly in this function, so that docstring is far away from this code. Maybe just add a comment where you mention the empty spaces like `see _get_class_with_reference docs about the space`. The params doc itself is a bit confusing (the phrasing looks like the space will cause the lines to be mixed instead of preventing it), what about:

```
    :param bool add_space:
        inform if the name generated will need to have an unicode space at the end (generated by |space|),
        this is necessary when the class is the only reference of an attribute to avoid this line
        to be mixed with the next line over the documentation.
```

Reply:
As we discussed on chat, commit 9dfefcd fix this issue.
(Removing the need of add_space or |space|)